src/mono/wasm/build/WasmApp.targets

```diff
@@ -277,7 +278,8 @@
     </ReadLinesFromFile>
 
     <PropertyGroup>
-      <EmccFlags>$(_DefaultEmccFlags) $(EmccFlags)</EmccFlags>
+      <EmccFlags Condition="'$(_WasmDevel)' == 'true'">-O0</EmccFlags>
+      <EmccFlags>$(EmccFlags) $(_DefaultEmccFlags) $(EmccFlags)</EmccFlags>
```

Comment:
This overwrites any previous set value for `EmccFlags`. Instead, if you want to prepend, then:

```xml
<EmccFlags>$(_DefaultEmccFlags) $(EmccFlags)</EmccFlags>
<EmccFlags Condition="'$(_WasmDevel)' == 'true'">-O0 $(EmccFlags)</EmccFlags>
```